kubernetes-on-ubuntu-vmss/azuredeploy.json

```diff
@@ -0,0 +1,398 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"_artifactsLocation": {
+			"type": "string",
+			"metadata": {
+				"description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+			},
+			"defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/kubernetes-on-ubuntu-vmss/"
+		},
+		"_artifactsLocationSasToken": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+			},
+			"defaultValue": ""
+		},
+		"vmName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of your VM master node."
+			}
+		},
+		"vmssName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of your VMSS cluster."
+			}
+		},
+		"location": {
+            "type": "string",
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS Code)

Reply:
good catch, fixed